pkgs/development/python-modules/stanza/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib,
+buildPythonPackage,
+python3Packages,
```

Comment:
```suggestion
```

This is strictly forbidden in pythonPackages. Please list the inputs individually.

Reply:
Thanks, did so.